client/src/components/Dropdown/index.tsx

```diff
@@ -1,6 +1,61 @@
 import React, { useRef, useState } from "react";
-
+import styled from "styled-components";
 import chevron from "icons/chevron.svg";
+import { theme } from "consts/theme";
+
+const DropDownContainer = styled.div`
+  position: relative;
+  width: fit-content;
+  height: 2rem;
+  margin-right: 0.3rem;
+  background-color: ${theme.dropdownContainer.backgroundColor};
+`;
+
+const DropDownStyle = styled.div`
+  border: 1px solid ${theme.dropodwnStyle.border};
```

Comment:
```suggestion
  border: 1px solid ${({ theme }) => theme.dropodwnStyle.border};
```

Slight difference, but this implementation:
```tsx
${ theme.foo }
```

will not change the actual color when the theme gets switched.
Imagine if the theme.foo is color "black". Users switches theme and theme.foo ends up being "white".
In this first example, the color wouldn't change.

But in this one:

```tsx
${({ theme }) => theme.foo}
```

it would.

Reply:
Because this - interpolation with function works better to rerender when theme changes.

There is a term for this: second interpolation is "reactive" and the first one isn't.

"reactive" = changes when an improtant value changes
"not-reactive" = doesn't change when an important value changes, stays fixed.